.agents/skills/playwright-best-practices/core/global-setup.md

```
@@ -0,0 +1,434 @@
+      },
+      dependencies: ['setup'],
+    },
+  ],
+})
+```
+
+## Parallel Execution Caveats
+
+### Understanding Global Setup Execution
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)